lib/gem_ext/gem_ext.rb

```diff
@@ -0,0 +1,10 @@
+require_dependency 'gem_ext/doorkeeper/application'
```

Comment:
I could not for the life of me avoid a dup enum warning without using the require_dependency on initialize. So that's why this is here...